src/components/LoginForm/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC, useEffect, useState } from 'react';
+import * as yup from 'yup';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Types_ActiveSession, Types_SessionForm } from '@customTypes/LoginTypes';
+import axios from 'axios';
+import { useRouter } from 'next/router';
+
+const LoginForm: FC = () => {
+  const router = useRouter();
+  let [sessionStatus, setSessionStatus] = useState<string|Types_ActiveSession>("");
+  
+  useEffect(()=> {
+    window.localStorage.setItem("localActiveSession", "no active session");
+  }, []);
+
+  const ValidationSessionForm_schema = yup.object().shape({
```

Comment:
@LilianaRestrepoTorres
This is the implementation of yup as validations schemes instead of having the complete regex wrote here with constraints and if's, making it easier to handle the errors and validations :D